api/swagger.yaml

```diff
@@ -5965,6 +5965,54 @@ paths:
           description: "Volume name or ID"
           type: "string"
       tags: ["Volume"]
+    put:
+      summary: "Update a volume"
+      operationId: "VolumeUpdate"
+      consumes: ["application/json"]
+      produces: ["application/json"]
+      responses:
+        200:
+          description: "The volume was updated successfully"
+          schema:
+            $ref: "#/definitions/Volume"
+        404:
+          description: "No such volume"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+        500:
+          description: "Server error"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+      parameters:
+        - name: "name"
+          in: "path"
+          required: true
+          description: "Volume name or ID"
+          type: "string"
+        - name: "volumeConfigUpdate"
+          in: "body"
+          required: true
+          description: "Volume configuration update"
+          schema:
+            type: "object"
+            properties:
+              DriverOpts:
+                description: "A mapping of driver options and values to change. These options are passed directly to the driver and are driver specific."
+                type: "object"
+                additionalProperties:
+                  type: "string"
+              Labels:
```

Comment:
DriverOpts and labels are not the same.  You are correct that DriverOpts are passed as -o <key=value>.  DriverOpts options are specific to the driver.  Labels are described [here](https://docs.docker.com/engine/userguide/labels-custom-metadata/).

Having said that, I should remove the labels from this work since they are really related to #25291.